c6021light/test/integration/StatelessRouting.cpp

```diff
@@ -86,6 +94,9 @@ TEST_P(TurnoutRoutingFixture, TurnoutRequest_I2CtoCANandLocoNet_WithGeneratedRes
       MarklinI2C::Messages::AccessoryMsg::makeOutbound(turnout, direction, power);
   EXPECT_CALL(i2cHal, sendI2CMessage(i2cResponseMessage));
   EXPECT_CALL(lnTx, DoAsyncSend(LnPacket));
+  EXPECT_CALL(xnHal, SetTrntPos(turnout.getNumericAddress().value() + 4,
```

Comment:
This is not exactly understandable code. Is there maybe a better abstraction?

Affects three lines in this file.